README.rst

```diff
@@ -1,106 +1,139 @@
-+------------------+-----------------------+
-
-**NOTE**: Since 6.2 version, eox-tenant does not support Django 2.2
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
++------------------+-----------------+
+| Open edX Release | Version         |
++==================+=================+
+| Ironwood         | < 3.0           |
++------------------+-----------------+
+| Juniper          | >= 3.0 < 4.0    |
++------------------+-----------------+
+| Koa              | >= 4.0 <= 5.1.3 |
++------------------+-----------------+
+| Lilac            | >= 4.0 < 6.2    |
++------------------+-----------------+
+| Maple            | >= 6.0          |
++------------------+-----------------+
+| Nutmeg           | >= 6.2          |
++------------------+-----------------+
+| Olive            | >= 8.0          |
++------------------+-----------------+
+| Palm             | >= v11.7.0      |
++------------------+-----------------+
+| Quince           | >= v11.7.0      |
++------------------+-----------------+
+| Redwood          | >= v11.7.0      |
++------------------+-----------------+
+
+⚠️ Since the 6.2 version, eox-tenant does not support Django 2.2
+
+The plugin is configured for the latest release (Redwood). The following changes in the plugin settings should be applied to be used for previous releases.
+
+**Maple**
+
+For version  11.X compatible
 
 .. code-block:: yaml
 
-  GET_BRANDING_API: 'eox_tenant.edxapp_wrapper.backends.branding_api_h_v1'
-  GET_CERTIFICATES_MODULE: 'eox_tenant.edxapp_wrapper.backends.certificates_module_i_v1'
-  GET_SITE_CONFIGURATION_MODULE: 'eox_tenant.edxapp_wrapper.backends.site_configuration_module_i_v1'
-  GET_THEMING_HELPERS: 'eox_tenant.edxapp_wrapper.backends.theming_helpers_h_v1'
-  EOX_TENANT_EDX_AUTH_BACKEND: "eox_tenant.edxapp_wrapper.backends.edx_auth_i_v1"
-  EOX_TENANT_USERS_BACKEND: 'eox_tenant.edxapp_wrapper.backends.users_i_v1'
-  EDXMAKO_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.edxmako_h_v1'
-  UTILS_MODULE_BACKEND: 'eox_tenant.edxapp_wrapper.backends.util_h_v1'
+  EOX_TENANT_EDX_AUTH_BACKEND = "eox_tenant.edxapp_wrapper.backends.edx_auth_i_v1"
+
+Those settings can be changed in ``eox_tenant/settings/common.py`` or, for example, in the instance settings.
+
+🚨 If the release you are looking for is not listed, please note:
 
-**Juniper**
+- If the Open edX release is compatible with the current eox-tenant version (see `Compatibility Notes <https://github.com/eduNEXT/eox-tenant?tab=readme-ov-file#compatibility-notes>`_), the default configuration is sufficient.
+- If incompatible, you can refer to the README from the relevant version tag for configuration details (e.g., `v6.2.0 README <https://github.com/eduNEXT/eox-tenant/blob/v6.2.0/README.rst>`_).
 
-For version >= 3.4
+🚨 For version > 10.0.0 you need to enable eox-tenant adding in the LMS configuration:
```

Comment:
Just one last comment. I'll be on the lookout to approve the PR
```suggestion
🚨 For version < 10.0.0 you need to enable eox-tenant adding in the LMS configuration:
```

Reply:
😱 Good catch thank you @BryanttV !!